src/main/kotlin/com/vanniktech/dependency/graph/generator/DependencyContainer.kt

```diff
@@ -0,0 +1,20 @@
+package com.vanniktech.dependency.graph.generator
+
+import org.gradle.api.Project as GradleProject
+import org.gradle.api.artifacts.ResolvedDependency as GradleResolvedDependency
+
+sealed interface DependencyContainer {
+
+  companion object {
```

Comment:
```suggestion
  internal companion object {
```

Reply:
Oups, unfortunately this is not valid code

> e: /home/runner/work/gradle-dependency-graph-generator-plugin/gradle-dependency-graph-generator-plugin/src/main/kotlin/com/vanniktech/dependency/graph/generator/DependencyContainer.kt: (8, 3): Modifier 'internal' is not applicable inside 'interface'